src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -550,6 +644,46 @@ public static ulong ToUInt64(ReadOnlySpan<byte> value)
             return Unsafe.ReadUnaligned<ulong>(ref MemoryMarshal.GetReference(value));
         }
 
+        /// <summary>
+        /// Returns a 128-bit unsigned integer converted from four bytes at a specified position in a byte array.
```

Comment:
8 bytes for long, so this would be 16 bytes?

Reply:
Yes. Thought I had fixed all these already.

It means the `ulong` one is also wrong and says `four` rather than `eight`

PRs are welcome!